novuRajan/gannt-chart
# Ac2 gantt 122 task and subtask deletion

**Description**

This pull request introduces the delete functionality for tasks and subtasks, ensuring the removal of dependencies wherever they exist.

**Changes**

Added a function to create a delete button, which is then generated and appended in the edit form.
Implemented a function to handle the deletion process.

**Testing Instructions**

To test the functionality, right-click on one of the taskbars. In the edit modal, a delete button has been added. Click the button and confirm whether or not to proceed with the deletion. Thoroughly test the feature to ensure that deletion actions work as expected. Perform regression testing to check for any unintended side effects.
